app/javascript/stylesheets/general.scss

```diff
@@ -0,0 +1,711 @@
+// scss
+@import "../../../node_modules/mappy-breakpoints/mappy-breakpoints.scss";
+
+$breakpoints: (
+  "small": 375px,
+  "large": 960px,
+);
+
+//////////////////////////////////////////////////////////////////////////
+//COLORS
+$primary-color: #6600cc;
+$dark-color: #252636;
+$white-color: #ffffff;
+$gray-color: #aaaaaa;
+$shadow-color: #f5f4f9;
+
+//////////////////////////////////////////////////////////////////////////
+//HOMEPAGE
+.hero-img {
+  background-image: linear-gradient(
+      rgba(37, 38, 54, 0.2),
+      rgba(37, 38, 54, 0.2)
+    ),
+    url("./../images/banner.svg");
+  height: 45rem;
+  background-position: left;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+  @include mappy-bp(small large) {
```

Comment:
**issue (blocking)**: this media query means the height 34rem will only be applied from small to large breakpoints, so on screens larger than 960px it will be 45rem again, is that the goal?
Even if that is the case this would be better if written in a mobile-first approach:
```scss
@include mappy-bp(small) { // applies from small up
    height: 34.375rem;
}
@include mappy-bp(large) { // applies from large up (will override small)
    height: 45rem;
}
```

Reply:
Does this mean, all the code should be inside includes?

Reply:
no, only the styles that should be applied from "small" up and then from "large" up.
Outside the includes will be code that applies to screens smaller than "small"